aspnetcore/fundamentals/servers/includes/memory-eviction2.md

```diff
@@ -0,0 +1,23 @@
+## Automatic eviction from memory pool
+
+The memory pools used by Kestrel, IIS, and HTTP.sys automatically evict memory blocks when the application is idle or under low load. The feature runs automatically and doesn't need to be enabled or configured manually.
+
+In versions of .NET earlier than 10, memory allocated by the pool remains reserved, even when not in use. This automatic eviction feature reduces overall memory usage and helps applications stay responsive under varying workloads.
+
+### Use memory pool metrics
+
+The default memory pool used by the ASP.NET Core server implementations includes metrics, which can be used to monitor and analyze memory usage patterns. The metrics are under the name `"Microsoft.AspNetCore.MemoryPool"`.
+
+For information about metrics and how to use them, see <xref:log-mon/metrics/metrics>.
+
+## Manage memory pools
+
+Besides using memory pools efficiently by evicting unneeded memory blocks, ASP.NET Core provides a built-in [IMemoryPoolFactory](https://source.dot.net/#Microsoft.AspNetCore.Connections.Abstractions/IMemoryPoolFactory.cs) and an implementation for it. It makes the implementation available to your application through dependency injection.
```

Comment:
```suggestion
Besides using memory pools efficiently by evicting unneeded memory blocks, ASP.NET Core provides a built-in [IMemoryPoolFactory](https://source.dot.net/#Microsoft.AspNetCore.Connections.Abstractions/IMemoryPoolFactory.cs) and an implementation for it. ASP.NET Core makes the implementation available to your application through dependency injection.
```
Minor clarification if you choose to add.  Not required.

Reply:
good catch thanks!